code/modules/surgery/organ_manipulation.dm

```diff
@@ -166,18 +160,12 @@
 
 /datum/surgery_step/manipulate_organs/success(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery, default_display_results)
 	if(current_type == "posibrain")
-		if(istype(tool, /obj/item/organ_storage))
-			tool.icon_state = "evidenceobj"
-			tool.desc = "A container for holding body parts."
-			tool.cut_overlays()
-			tool = tool.contents[1]
-
-		user.temporarilyRemoveItemFromInventory(tool)
-		spawn(1)
-			I = new /obj/item/organ/brain/mmi_holder/posibrain(tool)
-			I.Insert(target)
-			user.visible_message("<span class='notice'>[user] inserts [tool] into [target]'s [parse_zone(target_zone)]!</span>",
-				"<span class='notice'>You insert [tool] into [target]'s [parse_zone(target_zone)].</span>")
+		user.temporarilyRemoveItemFromInventory(tool, TRUE)
+		I = new /obj/item/organ/brain/mmi_holder/posibrain(null, tool)
```

Comment:
What is 'I' and also where is it even defined

Reply:
It's defined on line 74 in this file
```
/datum/surgery_step/manipulate_organs
	time = 64
	name = "manipulate organs"
	repeatable = TRUE
	implements = list(/obj/item/organ = 100, /obj/item/organ_storage = 100, /obj/item/mmi = 100)
	var/implements_extract = list(TOOL_HEMOSTAT = 100, TOOL_CROWBAR = 55, /obj/item/kitchen/fork = 35)
	var/current_type
	var/obj/item/organ/I = null
```

Reply:
Could this possibly be renamed to something more descriptive?